analyses/cnv-chrom-plot/gistic_plot.Rmd

```diff
@@ -210,7 +207,7 @@ lapply(unique(seg_data$short_histology), function(histology_group) {
     )
 
   # Save the plot to a png
-  # Have to ouse @ggplot to extract the portion of the ggbio object with the plot
+  # Have to use @ggplot to extract the portion of the ggbio object with the plot
   ggplot2::ggsave(
     plot = hist_plot@ggplot,
     filename = file.path(plots_dir, paste0(histology_group, "_plot.png")),
```

Comment:
On lines 219-220 below, you are printing the `hist_plot`. However, when rendered there are no plots displayed here. I am thinking you may not want this here as it looks like there would be a substantial amount of plots displayed in the notebook.